### ISSUE TITLE
Questions about reactor's new logback AsyncAppender implementation.

### ISSUE THREAD

**Issue by bobwenx** [reactor/reactor#477]
The Logback `AsyncAppender`'s implementation has changed since `2.0.2.RELEASE`. i have one question about this new implementation:
Are `AsyncAppender` automatically clear `MutableSignal.value`( contains object reference to `ILoggingEvent`)  after loggingEvent be writer to delegated appender?
```
@Override
public void onNext(ILoggingEvent iLoggingEvent) {
	aai.appendLoopOnAppenders(iLoggingEvent);
    // after method end, does MutableSignal.value (reference to this iLoggingEvent) get cleared?
}
   
```
Or will it just holding a reference of `ILoggingEvent` until buffer get overwrite(default backlog set be 1024 * 1024)?.
if it is the later case, it will bring problems in my project. because iLoggingEvent contains some argument object(used to format logger string), and these objects will be holding in RingBuffer until RingBuffer's cursor get overwrite.

**Comment by smaldini** [comment]
It is indeed the later, we noticed that and we will provide an alternative, most likely switch to RingBufferWorkProcessor which clears values.

**Comment by smaldini** [comment]
2.0.3 won't be far away

**Comment by bobwenx** [comment]
Thanks @smaldini.

**Comment by smaldini** [comment]
Fixed in 2.0.3.RELEASE now out.
